### PR TITLE
Adapt configuration to be able to use Heroku.

### PR DESCRIPTION
**Pull request by calavera**
Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -68,7 +67,9 @@ func NewAPI(config *conf.Config, db *gorm.DB, proxy payerProxy) *API {
 }
 
 func (a *API) Serve() error {
-	return http.ListenAndServe(fmt.Sprintf(":%d", a.port), a.handler)
+	l := fmt.Sprintf(":%d", a.port)
+	logrus.Infof("Netlify Subscriptions API started on: %s", l)
```
Comment (rybit) [inline api/api.go]: use the logger on the api. `a.log.Infof(`

File: conf/configuration.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
+package conf
+
+import (
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+
+	//"github.com/benoitmasson/viper"
```
Comment (rybit) [inline conf/configuration.go]: kill this

File: conf/configuration.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+	if config.Port == 0 && os.Getenv("PORT") != "" {
+		port, err := strconv.Atoi(os.Getenv("PORT"))
+		if err != nil {
+			return nil, errors.Wrap(err, "formatting PORT into int")
+		}
+
+		config.Port = port
+	}
+
+	if config.Port == 0 {
```
Comment (rybit) [inline conf/configuration.go:91]: I don't think this is what you want. If you don't specify a port it should be valid b/c the system will pic an available one.

**Review by rybit — COMMENTED** [review]
little nit picks ~ quick fix then lgtm

File: conf/configuration.go (hunk 3 above)
Comment (calavera) [reply conf/configuration.go:91]: which one though? I rather make it explicit, so you know which port it is by default.

File: api/api.go (hunk 1 above)
Comment (calavera) [reply api/api.go]: done

File: conf/configuration.go (hunk 2 above)
Comment (calavera) [reply conf/configuration.go]: done

**Pull request merged**